bindings/rust/bench/benches/handshake.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use bench::{S2nTls, TlsBenchHarness};
```

Comment:
Nit: Maybe a better name for the "benches" folder would be "scenarios"?

Reply:
"benches" is the standard name for the folder containing benchmarks for rust. With "scenarios," I think it also isn't immediately obvious that the point is to run benchmarks as opposed to running tests/examples